pkg/test/e2e/nodeport-proxy/const.go

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2020 The Kubermatic Kubernetes Platform contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package nodeportproxy
+
+// TODO(irozzo) make registries configurable
+const (
+	AgnosImage   = "k8s.gcr.io/e2e-test-images/agnhost:2.21"
+	NetexecImage = "gcr.io/kubernetes-e2e-test-images/netexec:1.1"
```

Comment:
Is there a specific use for using those two different images?
The commenter [here](https://github.com/kubernetes/kubernetes/issues/76342#issuecomment-495641423) says that the 1.1 version already contains netexec.

Reply:
Good catch, the `agnhost` image contains all that we need for the test.